tests/pools/test_pool_rpc.py

```diff
@@ -402,7 +402,7 @@ def mempool_empty() -> bool:
                                 [some_wallet.wallet_state_manager.private_key], status.current.owner_pubkey
                             )
                             assert owner_sk is not None
-                            assert owner_sk != auth_sk
+                            assert owner_sk[0] != auth_sk
```

Comment:
`owner_sk` is a pair of (secret_key, index). This test was meant to compare the secret keys

Reply:
Wouldn't have caught the error here, but a not-tuple return would be nice.  Not for this PR though.

Reply:
yeah, I'm not entirely confident how this is related to `blspy` types, but it seems that untyped variables are "viral", so even comparing `Tuple[PrivateKey, int]` to `PrivateKey` doesn't trigger mypy, because `PrivateKey` is untyped

Reply:
We have strict equality enabled for the strict files, but not for the files in the mypy exceptions list.  But yes, if one thing is an `Any` then...  almost anything you do with it is `Any` as well and mypy won't complain much.

https://github.com/Chia-Network/chia-blockchain/blob/f0cb47baefbab6f0e94bb60dd8df9985bf4079dd/mypy.ini.template#L17